<?xml version="1.0"?>
<launch>
  <arg name="boo_control" default="true"/>
  <arg name="boo_thresh_lo" default="0.20"/>
  <arg name="boo_thresh_hi" default="0.22"/>

  <node ns="/" pkg="dataspeed_can_usb" type="can_node" name="can_node" output="screen">
    <remap from="can_bus_1" to="can_bus_1"/>
    <param name="bitrate_1" value="0" />
    <remap from="can_bus_2" to="can_bus_2"/>
    <param name="bitrate_2" value="0" />
    <remap from="can_bus_3" to="can_bus_dbw"/>
    <param name="bitrate_3" value="500000" />
    <param name="channel_3_mask_0"  value="0x800007F0" />
    <param name="channel_3_match_0" value="0x00000060" /> <!-- SID 0x060 to 0x06F -->
    <param name="channel_3_mask_1"  value="0x800007FC" />
    <param name="channel_3_match_1" value="0x00000070" /> <!-- SID 0x070 to 0x073 -->
    <remap from="can_bus_4" to="can_bus_4"/>
    <param name="bitrate_4" value="0" />
  </node>

  <node ns="vehicle" pkg="dbw_mkz_can" type="dbw_node" name="dbw_node" output="screen">
    <remap from="can_rx" to="/can_bus_dbw/can_rx"/>
    <remap from="can_tx" to="/can_bus_dbw/can_tx"/>
    <param name="boo_control" value="$(arg boo_control)" />
    <param name="boo_thresh_lo" value="$(arg boo_thresh_lo)" />
    <param name="boo_thresh_hi" value="$(arg boo_thresh_hi)" />
  </node>

</launch>
